packages/client/src/components/Footer/Footer.component.js

```diff
@@ -4,43 +4,57 @@ import { Link } from 'react-router-dom';
 
 export const Footer = () => {
   return (
-    <footer className="footer-container">
-      <div className="footer-content-section">
-        <div>
-          <Link to="/">
-            <p className="logo-text">KAOSPILOT</p>
-          </Link>
+    <footer>
+      <div className="footer-bg">
+        <div className="footer-container">
+          <div className="copyright-disclaimer-container">
+            <p className="copyright-disclaimer-text">
+              Copyright Kaospilot. All methods & tools © their respective
+              owners.
+            </p>
+            <p className="copyright-disclaimer-text">
+              Are you an owner to a method/tool, please contact us.
```

Comment:
Is it worth to add a link here to maybe a mailto?

Reply:
Agree with Milton on that. Maybe a mailto linked to the 'contect us' could be a good idea

Reply:
Done✨